pdfminer/pdfpage.py

```diff
@@ -38,23 +39,27 @@ class PDFPage:
       rotate: the page rotation (in degree).
       annots: the page annotations.
       beads: a chain that represents natural reading order.
+      label: the page's label (typically, the logical page number).
     """
 
     def __init__(
         self,
         doc: PDFDocument,
         pageid: object,
-        attrs: object
+        attrs: object,
+        label: Optional[str]
```

Comment:
Adding a default `None` value makes code below simpler.

Reply:
I don't think it does, but I've cleaned up the callsites quite a bit another way. Please reopen if I missed something.